public/javascripts/addusercomponents.js

```diff
@@ -43,24 +43,26 @@ var UserList = React.createClass({
 			}.bind(this)
 		});
 	},
+	handleAdd: function(username) {
+		console.log(username);
+	},
 	render: function() {
-		var boardId = this.props.boardId;
```

Comment:
What are the benefits of taking this out and writing out `this.props.boardId` in each subfunction?